src/osd/PG.cc

```diff
@@ -4262,6 +4263,11 @@ void PG::scrub_process_inconsistent()
 	  }
 	}
       }
+
+      scrubber.authoritative.clear();
+      scrubber.inconsistent.clear();
+      scrubber.missing.clear();
```

Comment:
if repair is false, we can't call scrub_process_inconsistent. For norepair, in scrub_finish, it will call scrub_clear_state and in this func it all scrubber.reset clear those map.
a)call scurb_process_inconsistent don't care repair.
b) or don't add info into those maps when repair is false. This can do in PGBackend::be_compare_scrubmaps(which already has a parameter repair).

@liewegas how to choose or have better method? Thanks!